sdk/notificationhubs/notification-hubs/src/auth/sasTokenCredential.ts

```diff
@@ -91,9 +41,13 @@ export class SasTokenProviderImpl implements SasTokenProvider {
 
   /**
    * Gets the sas token for the specified audience
-   * @param audience - The audience for which the token is desired.
+   * @param scopes - The audience for which the token is desired.
    */
-  async getToken(audience: string): Promise<AccessToken> {
+  async getToken(
+    scopes: string | string[],
+    _options?: GetTokenOptions
```

Comment:
can we remove the options parameter if it is not being used?

Reply:
Removed